snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotStore.java

```diff
@@ -512,19 +512,8 @@ FileBasedSnapshot persistNewSnapshot(
       moveToSnapshotDirectory(directory, destination);
 
       final var checksumPath = buildSnapshotsChecksumPath(snapshotId);
-      final SfvChecksum actualChecksum;
       try {
-        // computing the checksum on the final destination also lets us detect any failures during
-        // the
-        // copy/move that could occur
-        actualChecksum = SnapshotChecksum.calculate(destination);
-        if (actualChecksum.getCombinedValue() != expectedChecksum) {
-          rollbackPartialSnapshot(destination);
-          throw new InvalidSnapshotChecksum(
-              directory, expectedChecksum, actualChecksum.getCombinedValue());
-        }
-
-        SnapshotChecksum.persist(checksumPath, actualChecksum);
+        SnapshotChecksum.persist(checksumPath, new SfvChecksum(expectedChecksum));
```

Comment:
:x: I think the SfvChecksum file will now have only the combined checksum, and not the per file checksum, right?

Reply:
I had no clue what this does 🤷🏼 👍🏼

Reply:
you want a quick sync on it? i didn't write it, but I wrote the issue on which it's based, so hopefully i should know :smile:

Reply:
But tbh I don't think that then this is the right approach to first skip this and then change the move. Because how would I know the checksum for all these files now? 

I think it makes more sense to directly write the files to the right location and build up the SfvChecksum via either the chunks (sent by the leader) or by reading the files (if we take the snapshot on our own).